webapp/src/components/App/Singlestudy/NavHeader.tsx

```diff
@@ -110,14 +116,19 @@ function NavHeader(props: Props) {
   const [deleteChildren, setDeleteChildren] = useState(false);
   const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
   const [openExportDialog, setOpenExportDialog] = useState(false);
+  const [isLatestVersion, setIsLatestVersion] = useState(
+    study?.version === latestVersion
+  );
   const { enqueueSnackbar } = useSnackbar();
   const enqueueErrorSnackbar = useEnqueueErrorSnackbar();
-  const isStudyFavorite = useAppSelector(isCurrentStudyFavorite);
-  const dispatch = useAppDispatch();
 
   const publicModeLabel =
     PUBLIC_MODE_LIST.find((mode) => mode.id === study?.publicMode)?.name || "";
 
+  useEffect(() => {
```

Comment:
`const isLatestVersion = study?.version === latestVersion;`

Because `latestVersion` re-render the component when it change (useAppSelector).